spec/features/idv/doc_auth/verify_info_step_spec.rb

```diff
@@ -75,7 +75,12 @@
 
   it 'allows the user to enter in a new ssn and displays updated info' do
     click_link t('idv.buttons.change_ssn_label')
+
     expect(page).to have_current_path(idv_ssn_path)
+    expect(
+      find_field(t('idv.form.ssn_label_html')).value,
+    ).to eq(DocAuthHelper::GOOD_SSN.gsub(/\D/, ''))
```

Comment:
Not clear to me why this `gsub` is needed. Are you expecting the field to be blank? Could you use GOOD_SSN_MASKED?

Reply:
I'm looking at the value of the field which is the SSN without the dashes.